src/views/settings/signature-settings.jsx

```diff
@@ -291,13 +309,16 @@ export default function SignatureSettings({
 									value: find(signatures, (signature) => signature.value.id === e.id).value.id
 								}
 							});
+							removeDarkThemeAttributeFromSignature();
```

Comment:
is there no other way to do it? It's not an ideal solution because I think you are un necessarily forcing the light theme even the theme is already light .Anyway it should not be called inside the oncChange

Reply:
did you tried the styled component with with white background as !important ?

Reply:
@geeky-abhishek I did not found any perfect solution. Because dark theme add extra attributes to div so it is over ride the css by default . So I have remove those dark theme attribute.

And onchange event we needs to call this function to remove that dark theme attribute because there is one case . Eg. If user does not have any signature then render signature is not in dom ( signature select from drop down does not have preview) . But if user add signature and select signatur on drop down then we have to show signatur preview in bottom of drop down. So that time div we needs to remove dark theme attributes . That's why added call on onchange